### DISCUSSION TITLE
Add a new language ​​to the multilingual model

### DISCUSSION THREAD

**Discussion by kdtiankong** [coqui-ai/TTS#1908] [General Q&A]
At present, multilingual seems to only support three languages. I want to add another language to the multilingual model. I don't know how to add it. Is there any relevant documentation? For example, I want to add Chinese, for exmple：

tts --model_name tts_models/multilingual/multi-dataset/your_tts --reference_wav /user/input.wav --speaker_wav /user/dest_output.wav --language_idx "zh-cn"

How can I add Chinese model, can anyone help me, thanks

**Comment by mbarnig** [comment]
I trained a multilingual model from scratch with 5 languages. I reported about it in [discussion #1727](https://github.com/coqui-ai/TTS/discussions/1727).

**Comment by wwt0805** [comment]
> At present, multilingual seems to only support three languages. I want to add another language to the multilingual model. I don't know how to add it. Is there any relevant documentation? For example, I want to add Chinese, for exmple：
> 
> tts --model_name tts_models/multilingual/multi-dataset/your_tts --reference_wav /user/input.wav --speaker_wav /user/dest_output.wav --language_idx "zh-cn"
> 
> How can I add Chinese model, can anyone help me, thanks

能否留个联系方式，交流一下

**Reply by mbarnig** [reply]
I reported in [discussion #1727](https://github.com/coqui-ai/TTS/discussions/1727) how to train a model with 5 languages.
